expts/main_run_multitask.py

```diff
@@ -29,21 +29,23 @@
 # WandB
 import wandb
 
+import torch; torch.set_float32_matmul_precision("medium")
```

Comment:
Optimization for e.g., A100s with tensor cores, doesn't require to lower precision to 16 in general.

Reply:
Is this a GPU specific command? Does it work when running on CPU? Is this the best place to have it, or perhaps in the `get_accelerator` if we find a GPU with mixed precision?

Reply:
I definitely agree. Had it there mostly to test it out. Moved it now as an optional feature to `load_accelerator` in the next commit.